### PR TITLE
Fix use of this in static method in Date component

### PR DESCRIPTION
**Pull request by jyash97**
`defaultQuery` was using a function `formatDate` with `this` keyword, which was throwing an error.

### CODE REVIEW

File: packages/web/src/components/date/DatePicker.js [hunk 1]
```diff
@@ -23,6 +22,7 @@ import Title from '../../styles/Title';
 import Flex from '../../styles/Flex';
 import CancelSvg from '../shared/CancelSvg';
 import { connect } from '../../utils';
+import formatDate from './utils';
```
Comment (metagrover) [inline packages/web/src/components/date/DatePicker.js]: We should put `formatDate` in core to enable reusability across native components as well.

@divyanshu013 what do you think?

Comment (divyanshu013) [reply packages/web/src/components/date/DatePicker.js]: Agreed bro :handshake:

Comment (metagrover) [reply packages/web/src/components/date/DatePicker.js]: 🤝

Comment (jyash97) [reply packages/web/src/components/date/DatePicker.js]: Need to move to `reactivecore` ?

Comment (metagrover) [reply packages/web/src/components/date/DatePicker.js]: yes, reactivecore utils.

**Pull request merged**